zkevm-circuits/src/witness/bytecode.rs

```diff
@@ -44,13 +44,17 @@ impl Bytecode {
 
         let mut push_data_left = 0;
         for (idx, byte) in self.bytes.iter().enumerate() {
-            let mut is_code = true;
-            if push_data_left > 0 {
-                is_code = false;
+            let is_code = push_data_left == 0;
+
+            if is_code {
+                let op = OpcodeId::try_from(*byte).unwrap();
+                if op.is_push() {
```

Comment:
@adria0 I am thinking that if we do not put the if op.is_push but make `push_data_left = op.len();` always, it will automatically work with all op codes. As for normal opcodes len returns 0.

Or am I missing something?